Intersect.Client/Entities/Animation.cs

```diff
@@ -109,40 +110,40 @@ public void Draw(bool upper = false, bool alternate = false)
 
             var rotationDegrees = 0f;
             var dontRotate = upper && MyBase.Upper.DisableRotations || !upper && MyBase.Lower.DisableRotations;
-            if ((AutoRotate || mRenderDir != -1) && !dontRotate)
+            if ((AutoRotate || mRenderDir != (Directions)(-1)) && !dontRotate)
```

Comment:
This condition literally does not make sense, and I mean it didn't make sense before you made changes.

The switch _only_ has cases present for 0-7, mRenderDir != -1 was either true, or it was false and did nothing. If mRenderDir was 8, 9, 10, 128937891237, it still wouldn't do anything.

I am not sure how to change this such that AutoRotate is actually relevant, because quite frankly from the contents of the switch, it just isn't.

Just change this to !dontRotate, the other conditions at present are totally useless (the switch itself validates, and does a better job validating, that mRenderDir is a usable value, since invalid values are just no-ops).

Reply:
btw ; before doing this, i tried with "default" and it lead to some funky behavior all over the place.

So; 

`if (AutoRotate || !dontRotate)`

or this? since AutoRotate is apparently not relevant (?)
`if (!dontRotate)`

(?)

Reply:
> AutoRotate is used by projectiles so we need it !
Ok, but what does it _do_? The switch inside the if always ignores it when the render direction is -1.

Reply:
> Ok, but what does it _do_? The switch inside the if always ignores it when the render direction is -1.

pretty sure that is related to this

![imagen](https://user-images.githubusercontent.com/17498701/217394987-2281bd1c-725b-47ac-8423-24b378e59b1f.png)